engine-tests/src/tests/pausable_precompiles.rs

```diff
@@ -0,0 +1,37 @@
+use crate::test_utils::{self, PAUSE_PRECOMPILES, RESUME_PRECOMPILES};
+use aurora_engine::parameters::PausePrecompilesCallArgs;
+use borsh::BorshSerialize;
+
+/// The ecrecover implementation in both the standalone and wasm contract should be the same.
+#[test]
+fn test_resume_precompiles() {
+    let mut runner = test_utils::deploy_evm();
+
+    let called_account_id = "aurora";
+    let call_args = PausePrecompilesCallArgs { paused_mask: 0b1 };
+
+    let mut input: Vec<u8> = Vec::new();
+    call_args.serialize(&mut input).unwrap();
+
+    let (outcome, error) = runner.call(RESUME_PRECOMPILES, called_account_id, input);
+
+    assert!(outcome.is_some(), "{:?}", outcome);
+    assert!(error.is_none(), "{:?}", error);
+}
+
+/// The ecrecover implementation in both the standalone and wasm contract should be the same.
+#[test]
+fn test_pause_precompiles() {
```

Comment:
It would be nice to test more than just "it didn't have an error during execution". I think the test should go something like 

1. Call pause precompiles
2. Try to execute a precompile -> it fails because they're paused
3. Call resume precompiles
4. Try to execute a precompile again -> it works

Reply:
Thanks for adding tests.

Yes, please increment the gas bounds of the failed tests. The purpose of these tests is to track how our gas usage changes over time. They fail for both increases and decreases to the gas cost. It allows us to make informed decisions about features like this. We need to answer the question of whether the feature we are adding is worth the additional cost in the case of increases. In this case I think it is clear that the pausing functionality it worth a few Tgas.

I think it's ok to share the standard_precompiles contract between tests because I think it is unlikely to change much over time.